views/random-beers.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="container">
+    {{#each beersFromRandom}}
+    <img class="beerimg1" src="{{this.image_url}}">
+       <p>{{responseFromAPI.name}}</p>
```

Comment:
On line 4-6, you used the wrong keyword so you're currently rendering half of the data.

It should be
```
 <p>{{this.name}}</p>
<p>{{this.description}}</p>
<p>{{this.tagline}}</p>